docs/ref/templates/builtins.txt

```diff
@@ -641,6 +657,17 @@ This example includes the contents of the template ``"foo/bar.html"``::
 
     {% include "foo/bar.html" %}
 
+You can use relative path (started from ``./``)
```

Comment:
I think if we want the parent directory, `../` would be okay to use (without a `./` prefix)